addons/base_import/models/base_import.py

```diff
@@ -637,6 +642,32 @@ def _parse_import_data_recursive(self, model, prefix, data, import_fields, optio
                 # We should be able to manage both case
                 index = import_fields.index(name)
                 self._parse_float_from_data(data, index, name, options)
+
+            elif field['type'] == 'binary' and field.get('attachment') and name in import_fields:
+                index = import_fields.index(name)
+                for num, line in enumerate(data):
+                    if re.match("^(http|https)://", line[index], re.IGNORECASE):
+                        try:
+                            with requests.get(line[index], stream=True) as response:
+                                maxsize = 2097152 #( IN BYTES 2MB )
+                                set_timeout = 120
+                                chunk_size = 0
+                                start = time.time()
+                                content = bytearray()
+                                response.raise_for_status()
+                                for chunk in response.iter_content(1024):
```

Comment:
Given the typical size of images, a more sensible chunk size would be 32k (32768)